core/src/main/java/edu/wpi/grip/core/http/PostHandler.java

```diff
@@ -0,0 +1,18 @@
+
+package edu.wpi.grip.core.http;
+
+/**
+ * Handler for converting incoming data over an HTTP {@code POST} operation.
+ */
+public interface PostHandler {
```

Comment:
`HttpSource` didn't originally implement this interface, but I needed a good way of removing the handler when the source gets removed from the pipeline. I could have a variable for this, but the current way is cleaner in my opinion